sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -119,13 +121,31 @@ describe.skip("Session Token", function (this: Suite) {
     // Create items using both clients so they each establish a session with the backend
     const container2 = clientB.database(dbId).container(containerId);
     await Promise.all([createItem(container), createItem(container2)]);
+  });
+
+  it("Should check if the MaxIntegratedCacheStaleness header is set.", async function () {
+    const dbId = addEntropy("maxIntegratedCacheTestDB");
+    const containerId = addEntropy("maxIntegratedCacheTestContainer");
+
+    const client = new CosmosClient({
+      endpoint,
+      key: masterKey,
+      consistencyLevel: ConsistencyLevel.Eventual,
+    });
+
+    // Create Database and Container
+    const { database } = await client.databases.createIfNotExists({
+      id: dbId,
+    });
+    const { container } = await database.containers.createIfNotExists({
+      id: containerId,
+    });
+    client.database(dbId).container(containerId);
```

Comment:
Result is not consumed, is this line required?

Reply:
not needed